src/app/views/experiment_page.js

```diff
@@ -0,0 +1,68 @@
+import View from 'ampersand-view';
+import dom from 'ampersand-dom';
+import mustache from 'mustache';
+import webChannel from '../lib/web_channel';
+
+export default View.extend({
+  template: mustache.render(`<section class="page" data-hook="experiment-page">
```

Comment:
FWIW, using `data-hook` to fill things in DOM-wise, I've been pretty happy [just using ES6 multiline template strings](https://github.com/lmorchard/c25k-web/blob/master/src/js/views/Workout.js)